lib/experimental/Navigation/Header/Header/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button } from "@/components/Actions/Button"
+import { IconType } from "@/components/Utilities/Icon"
+import AlignTextJustify from "@/icons/AlignTextJustify"
+import { cn } from "@/lib/utils"
+import Breadcrumbs, { type BreadcrumbItemType } from "../Breadcrumbs"
+
+import {
+  ModuleAvatar,
+  type IconType as ModuleAvatarIconType,
+} from "@/experimental/Information/ModuleAvatar"
+
+type HeaderProps = {
+  module: {
+    name: string
+    href: string
+    icon: ModuleAvatarIconType
+  }
+  tree?: BreadcrumbItemType[]
+  actions?: {
+    label: string
+    icon: IconType
+    onClick: () => void
+  }[]
+  menu: {
+    show: boolean
+    onClick: () => void
+  }
```

Comment:
Maybe we should be more prescriptive in terms of what actions do we allow here? I thought we wanted to encourage "settings" and "help" or somethin like that. Do we want to go specific, or give more freedom to devs? I'm afraid of abuse.

Reply:
I reviewed this with Mario this morning, and it's quite open. Actions aren't limited to just a few, they can be anything to support every case we have at Factorial.